okon_actions_example.py

```diff
@@ -0,0 +1,53 @@
+""" Script for showing usage of actions defined in okon_acions module.
+
+Script prepared for testing actions defined in okon_actions module
+in OKON simulation (simulation-2.3dev version).
+
+Run this script after starting the simulation.
+"""
+import time
+
+import py_trees
+
+from okon_actions import Rotate, SetDepth
+from okon_client import OkonClient
+
+oc = OkonClient(ip="127.0.0.1", port=44210, sync_interval=.05, debug=False)
+oc.connect()
+time.sleep(1.)
+
+def create_root():
+    """ description
+
+    Long description
+    """
+    root = py_trees.composites.Sequence("Sequence")
+    set_depth_action_1 = SetDepth(name = "Set Depth to 0.2 m",okon=oc.okon, depth=0.2, delta=0.005)
+    rotate_action_2 = Rotate(name = "Turn left",okon=oc.okon, add_angle=-45., delta=1.)
+    set_depth_action_3 = SetDepth(name = "Set Depth to 0.8 m",okon=oc.okon, depth=0.8, delta=0.005)
+    rotate_action_4 = Rotate(name = "Turn right",okon=oc.okon, add_angle=45., delta=1.)
+    root.add_children([set_depth_action_1, rotate_action_2, set_depth_action_3, rotate_action_4])
+    return root
+
+
+def main():
+    """ Main function of the script"""
+    py_trees.logging.level = py_trees.logging.Level.DEBUG
+
+    root = create_root()
+    root.setup_with_descendants()
+    
+    for i in range(0, 30):
+        try:
+            tick_string = "Tick"
+            print(f"\n{tick_string:.>25} {i :.<25}\n")
```

Comment:
Today I learnt about f-string nesting, symetrical filling and debugging 😄. Check this out 🤓 
```suggestion
    for tick in range(0, 30):
        try:
            print(f"\n{f'{tick = }':^25}\n")